templates/chat-room/src/client.tsx

```diff
@@ -0,0 +1,20 @@
+import "./styles.css";
```

Comment:
If we're moving all the server stuff to "party", then let's rename "src" to "app"?